packages/storage-web-api/src/request-validators/submit-website-scan-request-validator.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as ApiContracts from 'api-contracts';
+import { Context } from '@azure/functions';
+import { inject, injectable } from 'inversify';
+import _ from 'lodash';
+import { GuidGenerator, ServiceConfiguration } from 'common';
+import { ApiRequestValidator, HttpResponse, WebApiErrorCodes } from 'service-library';
+import * as cronParser from 'cron-parser';
+
+@injectable()
+export class SubmitWebsiteScanRequestValidator extends ApiRequestValidator {
```

Comment:
Similar to controller comment we can either use HTTP verbs or DB CRUD naming convention.